src/PostgREST/QueryBuilder.hs

```diff
@@ -105,6 +108,10 @@ mutateRequestToQuery (Update mainQi uCols logicForest returnings) =
         ]
   where
     cols = intercalate ", " (pgFmtIdent <> const " = _." <> pgFmtIdent <$> S.toList uCols)
+    empty_body_returned_columns :: SqlFragment
+    empty_body_returned_columns
+      | null returnings = "NULL"
+      | otherwise       = intercalate ", " (pgFmtColumn (QualifiedIdentifier mempty $ qiName mainQi) <$> returnings)
```

Comment:
This fragment is looking similar to `returned_columns` in `requestToCallProcQuery`. Do you think that could be factored in a function inside [QueryFragment.hs](https://github.com/PostgREST/postgrest/blob/master/src/PostgREST/Private/QueryFragment.hs)?

Reply:
Everything else is looking good. I was seeing if there could be some other way to do the fix but your solution is the simplest one.

Reply:
Can be done later. Thanks for the fix!